app/assets/stylesheets/button-group-custom.scss

```diff
@@ -0,0 +1,13 @@
+.govuk-button-group-custom {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 15px;
+  }
+  
+  @media (min-width: 40.0625em) {
```

Comment:
Is it possible to use `$govuk-breakpoints` instead of a fixed value?

Reply:
@psweeting1 - agreed. Have added that in now 👍🏽